Validators (operators) using systemd (#230)
moondance-labs/tanssi-docs: node-operators/validators/onboarding/index.md

---
title: Run a Validator Node
description: Learn how to run a validator (also called operators) to verify all the Tanssi-powered networks' transactions, providing economic security while earning rewards.
icon: octicons-arrow-up-right-24
template: index-page.html
---

# Onboarding

Your journey as a validator begins here. This section covers the foundational steps to set up your node and prepare your account for active participation in the Tanssi network.

> Use the following resources to get started as a validator:

Reviewer: ```suggestion
Use the following resources to understand the requirements before running a validator:
```

Just a minor nit 😬
